groups/rds/rds.tf

```diff
@@ -0,0 +1,130 @@
+# ------------------------------------------------------------------------------
+# RDS Security Group and rules
+# ------------------------------------------------------------------------------
+module "rds_security_group" {
+
+  source  = "terraform-aws-modules/security-group/aws"
+  version = "~> 3.0"
+
+  name        = "sgr-smartview-rds-001"
+  description = "Security group for the smartview RDS database"
+  vpc_id      = data.aws_vpc.vpc.id
+
+  ingress_cidr_blocks = concat(local.admin_cidrs, var.rds_onpremise_access)
+  ingress_rules       = ["oracle-db-tcp"]
+  ingress_with_cidr_blocks = [
+    {
+      from_port   = 5500
+      to_port     = 5500
+      protocol    = "tcp"
+      description = "Oracle Enterprise Manager"
+      cidr_blocks = join(",", concat(local.admin_cidrs, var.rds_onpremise_access))
+    }
+  ]
+  ingress_with_source_security_group_id = []
+
+  egress_rules = ["all-all"]
+}
+
+# ------------------------------------------------------------------------------
+# RDS Instance
+# ------------------------------------------------------------------------------
+module "smartview_rds" {
+  source  = "terraform-aws-modules/rds/aws"
+  version = "2.23.0" # Pinned version to ensure updates are a choice, can be upgraded if new features are available and required.
+
+  create_db_parameter_group = "true"
+  create_db_subnet_group    = "true"
+
+  identifier                 = join("-", ["rds", var.identifier, var.environment, "001"])
```

Comment:
May want to make this notation consistent with what's in line 63
e.g.:
```terraform
final_snapshot_identifier = "${var.identifier}-final-deletion-snapshot"
```

Reply:
These both use established naming schemes and so won't be amended here, I'm afraid